plugins.txt

```diff
@@ -112,20 +112,20 @@ mina-sshd-api-common:2.11.0-86.v836f585d47fa_
 mina-sshd-api-core:2.11.0-86.v836f585d47fa_
 node-iterator-api:55.v3b_77d4032326
 okhttp-api:4.11.0-157.v6852a_a_fa_ec11
-opentelemetry:2.17.0
-pipeline-build-step:505.v5f0844d8d126
+opentelemetry:2.18.0
```

Comment:
https://github.com/jenkinsci/opentelemetry-plugin/releases/tag/opentelemetry-2.18.0-beta1 => looks like tag and release were created but not the changelog notes